component/azstorage/datalake.go

```diff
@@ -578,6 +611,18 @@ func (dl *Datalake) TruncateFile(name string, size int64) error {
 	return dl.BlockBlob.TruncateFile(name, size)
 }
 
+func (dl *Datalake) getAccessControl(name string) (string, error) {
```

Comment:
Can this be renamed to getACL as it sounds confusing with the SDK's GetAccessControl method.

Reply:
updated